lib/subghz/protocols/secplus_v2.c

```diff
@@ -151,7 +151,7 @@ static bool subghz_protocol_secplus_v2_mix_order_decode(uint8_t order, uint16_t
     case 0x06: // 0b0110  2, 1, 0],
     case 0x09: // 0b1001  2, 1, 0],
         p[2] = a;
-        p[1] = b;
+        // p[1]: no change
```

Comment:
here you can just comment out the lines so that PVS does not swear, otherwise the readability of the code suffers

Reply:
I believe that my version is more clear because a valid C code that was commented out is confusing (why was it commented out? did someone forget to uncomment it? is this code a work in progress? etc)